### PR TITLE
update license to a align with CrowdStrike legal

### PR DESCRIPTION
**Pull request by domdepasquale**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,24 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright The Helm Authors.
```
Comment (redhatrises) [inline LICENSE:190]: Probably should make this the `Falcon Operator` Authors

File: hack/boilerplate.go.txt [hunk 2]
```diff
@@ -1,26 +1,204 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright The Helm Authors.
```
Comment (redhatrises) [inline hack/boilerplate.go.txt:191]: Probably should make this the `Falcon Operator` Authors

**Review by isimluk — APPROVED** [review]

**Comment by redhatrises** [comment]
Merging since this PR has been out for a while with no comments, and this license is needed for cert efforts which are going on right now.

**Comment by domdepasquale** [comment]
@mrcloudninja,

This is @domdepasquale from the Operator open source project.  We are changing the license governing this project to the Apache License v2.0 and as a contributor to this project, we would like to obtain your consent to this license change.  Do you consent to this license change?

**Pull request merged**